### PR TITLE
Updating README file 

### PR DESCRIPTION
**Pull request by ipekakyol**
Includes:

- updates to the README file
- updates to data access calls (Done by Leo)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,41 @@
 # GroupedUp
 
 This is the repository for group 47 in tutorial 0201. 
-The purpose of our application is too create an app for UofT students to find groups! There are 8 features: login/logout, create group, edit your user or group profile, Apply/Leave/Cancel/Approve group application, and find groups through our Matching Algorithm.
+The purpose of our application is too create an app for UofT students to find groups! There are 8 features: login/logout, create group, edit user or group profile, Apply/Leave/Cancel/Approve group application, and find groups through our Matching Algorithm.
```
Comment (aaryavatsa) [inline README.md]: > purpose of our application is too create

should be **to**, not too

also, is Matching Algorithm capitalized? someone pitch in.

**Review by aaryavatsa — CHANGES_REQUESTED** [review]
Looking great, just fix the one grammar error so we don't have to look back at this. Nice job on commenting the run instructions!

File: README.md (hunk 1 above)
Comment (ipekakyol) [reply README.md]: Thanks for catching that, just updated it

**Review by ileuxali — APPROVED** [review]
grammar changed :^)

**Review by aaryavatsa — APPROVED** [review]
thanks for fixing! everything looks good now

**Review by SharonHsieh22 — APPROVED** [review]
Thanks for typing this up!

**Pull request merged**